packages/babel-plugin-transform-jsxtreme-markdown/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 Any version not listed here is just a non-disruptive update in the `@mapbox/jsxtreme-markdown` dependency.
 
+## 0.6.0
```

Comment:
Should we bump versions to 1.0.0? Each of these packages has been stable for a long time now.